cliutil/utils.go

```diff
@@ -1,3 +1,6 @@
+// Package cliutil provides utility functions shared by multiple main packages.
+// TODO: createAndWriteEthKeyFile and GetEthereumPrivateKey are only used by swapd,
+// so we should move them out of here.
```

Comment:
I think these were here, because the recoverer called `GetEthereumPrivateKey`. Now that it is gone, we could place them in swapd's main package. Or if you think they'll be reused, I'll just delete the TODO.